src/workerd/io/BUILD.bazel

```diff
@@ -332,13 +335,17 @@ kj_test(
 
 kj_test(
     src = "promise-wrapper-test.c++",
-    deps = [":io"],
+    deps = [
+        ":io",
+        "//src/workerd/api:encoding",
```

Comment:
Neither of these two tests depends directly on encoding right? Let's IWYU here and leave these as they were before